lib/lsperfm/core/stats.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: utf-8
+require "thread"
+
+Thread.abort_on_exception = true
+
+module LogStash::PerfM
```

Comment:
Complicated to decide here, I'm actually not a big fan of long namespaces. Do you have a proposal in mind?